### PR TITLE
First draft of a Mission Statement

### PR DESCRIPTION
**Pull request by renrutnnej**
Adding my first pass at a Mission Statement after a great conversation with @gr2m yesterday. ☺️

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 :rocket: Welcome to the home of the Hoodie Editorial Team! :rocket:
 
+## Mission Statement
+
+The Hoodie Editorial Team exists to support [Hoodie’s Initiatives](http://hood.ie/initiatives/) ([Offline First](http://hood.ie/initiatives/#offline-first), [noBackend](http://hood.ie/initiatives/#nobackend), [Hoodie is Dreamcode](http://hood.ie/initiatives/#dreamcode), [tools](http://hood.ie/initiatives/#tools) and products, and encouraging [Non-Coding Experts in Open Source](http://hood.ie/initiatives/#noncoders)) through writing documentation, writing content for a website or blog to design, editing, public relations, social media and much more.
```
Comment (gr2m) [inline README.md]: Love it <3

As I read this, I wonder if we can change "Non-Coding Experts in Open Source" to "Non-Coding Contributors in Open Source", also on http://hood.ie/initiatives/#noncoders? Any thoughts?

Comment (renrutnnej) [inline README.md]: Ooh good point, I like that better. :+1:

File: README.md [hunk 2]
```diff
@@ -12,7 +18,6 @@ We:
 - **Plan and write blog posts** on or around Hoodie, Hoodie releases, side projects, TGIF, community member profiles, conference and meetup recaps, etc. :memo:
 - **Solicit and curate contributions** to our blog from the larger community. :mag_right:
 - **Plan and manage our Twitter account**, announce new content, help funnel support requests to the engineering team.
-- **Help co-ordinate The Hoodie Letter and Offline First newsletters.**
```
Comment (Charlotteis) [inline README.md]: Are Hoodie no longer doing newsletters?

Comment (renrutnnej) [inline README.md]: I omitted that because I've been curating and coordinating those on my own so they're no longer an Editorial Team responsibility.

**Comment by Charlotteis** [comment]
Looks good to me. Sad to see that we're no longer doing newsletters though.

**Comment by renrutnnej** [comment]
It's been 72 hours so I'm merging this now :+1:

**Pull request merged**